index.bs

```diff
@@ -402,9 +565,29 @@ perform the following steps:
 
 </div>
 
-Every [=bounce tracking timer period=] the user agent should run the
-[=bounce tracking timer=] algorithm given the [=wall clock=]'s
-[=wall clock/unsafe current time=].
+This algorithm is called from [=process response received for bounce tracking=],
+after a timeout of [=client bounce detection timer period=], unless it is cancelled
+by toggling [=bounce tracking record/client bounce detection running=].
```

Comment:
It's also called from 'process navigation start' if the navigation was caused by user activation. I think the semantics are really that it's called at the end of an 'extended navigation'?

Reply:
Yep, thanks for the correction, rewrote this section to clarify.